gradle/libs.version.toml

```diff
@@ -0,0 +1,35 @@
+[versions]
+android = "7.0.4"
+arrow = "1.0.1"
+arrowGradleConfig = "0.6.0"
+coroutines = "1.6.0"
+dokka = "1.6.10"
+gradlePublish = "0.20.0"
+javierscSemverGradlePlugin = "0.1.0-alpha.10"
+kotlin = "1.6.10"
+nexusPublish = "1.1.0"
+spotless = "6.2.0"
+
+[libraries]
+android = { module = "com.android.tools.build:gradle", version.ref = "android" }
+arrow-core = { module = "io.arrow-kt:arrow-core", version.ref = "arrow" }
+arrow-fxCoroutines = { module = "io.arrow-kt:arrow-fx-coroutines", version.ref = "arrow" }
+coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "coroutines" }
+dokka-base = { module = "org.jetbrains.dokka:dokka-base", version.ref = "dokka" }
+dokka-core = { module = "org.jetbrains.dokka:dokka-core", version.ref = "dokka" }
+dokka-testApi = { module = "org.jetbrains.dokka:dokka-test-api", version.ref = "dokka" }
+dokka-baseTestUtils = { module = "org.jetbrains.dokka:dokka-base-test-utils", version.ref = "dokka" }
+dokka-gfmPlugin = { module = "org.jetbrains.dokka:gfm-plugin", version.ref = "dokka" }
+dokka-gradlePlugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "dokka" }
+gradle-publishPlugin = { module = "com.gradle.publish:plugin-publish-plugin", version.ref = "gradlePublish" }
+gradleNexus-publishPlugin = { module = "io.github.gradle-nexus:publish-plugin", version.ref = "nexusPublish" }
+javiersc-semver-semverGradlePlugin = { module = "com.javiersc.semver:semver-gradle-plugin", version.ref = "javierscSemverGradlePlugin" }
+kotlin-gradlePlugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+kotlin-scriptingJsr223Unshaded = { module = "org.jetbrains.kotlin:kotlin-scripting-jsr223-unshaded", version.ref = "kotlin" }
+spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless" }
+
+[plugins]
+arrow-gradleConfig-formatter = { id = "io.arrow-kt.arrow-gradle-config-formatter", version.ref = "arrowGradleConfig" }
+arrow-gradleConfig-versioning = { id = "io.arrow-kt.arrow-gradle-config-versioning", version.ref = "arrowGradleConfig" }
+dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
+kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

Comment:
you're missing kapt unless this is not needed anymore

Reply:
arrow proper is using ksp?